src/components/CollectiveCard.js

```diff
@@ -67,28 +68,37 @@ class CollectiveCard extends React.Component {
     const backgroundImage = imagePreview(
       collective.backgroundImage ||
         get(collective, 'parentCollective.backgroundImage'),
-      null,
+      defaultBackgroundImage['COLLECTIVE'],
       { width: 400 },
     );
+
     if (!coverStyle.backgroundImage && backgroundImage) {
       coverStyle.backgroundImage = `url('${backgroundImage}')`;
       coverStyle.backgroundSize = 'cover';
       coverStyle.backgroundPosition = 'center center';
     }
 
+    const logoImage = imagePreview(
+      collective.image,
+      defaultImage['COLLECTIVE'],
+      { height: 65 },
+    );
+
     const truncatedDescription =
       (collective.description && firstSentence(collective.description, 80)) ||
       (collective.longDescription &&
         firstSentence(collective.longDescription, 80));
     const description = collective.description;
 
-    let route = this.props.collective.path || `/${this.props.collective.slug}`;
+    const params = {
+      slug: collective.slug,
+    };
     if (LoggedInUser) {
-      route += `?referral=${LoggedInUser.CollectiveId}`;
+      params.referral = LoggedInUser.CollectiveId;
     }
 
     return (
-      <Link route={route} target="_top">
+      <Link route="collective" target="_top" params={params} passHref>
```

Comment:
For clarification, if the direct child of a `Link` component from NextJS is an anchor tag, then `passHref` is not needed. If it is an abstraction over an anchor tag, like `StyledLink` then `passHref` is needed. Removed `passHref` here because the `Link` internal component automatically includes an anchor tag as the direct child of a NextJS Link.